frontends/web/src/routes/account/manage/components/coin-dropdown.tsx

```diff
@@ -0,0 +1,151 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { h, RenderableProps } from 'preact';
+import * as accountApi from '../../../../api/account';
+import { Select } from '../../../../components/forms';
+import { translate, TranslateProps } from '../../../../decorators/translate';
+
+// copied over from old addacconut.tsx
+const COIN_AND_ACCOUNT_CODES = {
```

Comment:
I will extend the 'supported-coins' endpoint to return not only the coincode, but also the name, then we can delete this code.

addaccount.tsx should probably be deleted completely, we will not be using it anymore, right?

Reply:
Done here: https://github.com/digitalbitbox/bitbox-wallet-app/pull/1253/